src/Config.ts

```diff
@@ -198,6 +198,16 @@ export interface ConfigParams {
    * @category Engine
    */
   matrixDetectionThreshold: number,
+  /**
+   * Sets the compatibility mode for behaviour of null value.
+   * If set, formula evaluating to null evaluates to 0 instead,
+   * as in Excel.
+   *
+   * @default false
+   *
+   * @category Engine
+   */
+  nullCompatibility: boolean,
```

Comment:
This option should be generic and not mention other product by name. It's not a compatibility but a vendor behavior. We need a better name

We can write a documentation or blog post how compatiblity with certaing product can be achived, but IMO it's just a matter of time when we see other products that evaluate empty cells to zero.

Reply:
any suggestions?

Reply:
maybe evaluateNullToZero ?

Reply:
does it reflect that it happens only *at the very end* of formula evaluation?